### ISSUE TITLE
Add support for .img files

### ISSUE THREAD

**Issue by openSourcerer9000** [jblindsay/whitebox-tools#147]
Any plans for adding support for .img files? I love TIF's as much as the next guy but in my state all the LiDAR is available as .img tiles. Having to convert hundreds of LiDAR files before I can use them isn't ideal.

**Comment by Afrancioni** [comment]
Yes there are always plans for this! If someone creates a Rust port of GDAL then we can certainly import that crate to make this feature request possible.

**Comment by alexbruy** [comment]
> If someone creates a Rust port of GDAL then we can certainly import that crate to make this feature request possible.

There is an ongoing effort to create Rust wrapper for GDAL [here](https://github.com/georust/gdal), maybe this is what you are looking for.

**Comment by jblindsay** [comment]
I've been aware of this project for some time now. I'd be very happy to move to a Rust port of GDAL for WBT; it would relieve me from an enormous amount of problematic code. However, I don't think a wrapper to GDAL is quite the solution that the project needs. Instead we need a native Rust solution. Of course, I suspect that it's unlikely to ever happen because GDAL is such an enormous amount of code.
